alembic/versions/809e29e0c0c7_gcn_aliases.py

```diff
@@ -0,0 +1,33 @@
+"""gcn_aliases
+
+Revision ID: 809e29e0c0c7
+Revises: fccc7e78c7aa
+Create Date: 2022-07-28 15:58:41.631666
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '809e29e0c0c7'
+down_revision = 'fccc7e78c7aa'
```

Comment:
Very likely to have to update the migration hash.

Reply:
@leoyang33 Did you check that the hash is up to date?

Reply:
I think this needs additional migrations for the columns you just added @leoyang33

Reply:
Im not sure how to do migrations

Reply:
Remove alembic/versions/809e29e0c0c7_gcn_aliases.py, git push and then commit the artifact migration in its place.